testsuite/MathMLToDisplay/General/Colors/issue1002.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+  <head>
+    <title>Test SVG output inherting color</title>
+    <!-- Copyright (c) 2015 MathJax Consortium
+         License: Apache License 2.0 -->
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <script type="text/javascript" src="../../../header.js"></script>
+    <script type="text/javascript" src="../../../scriptTests.js"></script>
+    <script type="text/javascript">
+      function getTestCases() {
+      var path = document.querySelectorAll('g')[0];
```

Comment:
Just FYI, `querySelectorAll()` is IE9+, so it would be better to use `document.getElementsByTagName("g")[0]` for this in most cases.  Because this test requires SVG, which is also IE9+, it doesn't matter here, but for tests that should run in all output modes, it would cause IE < 9 to crash.  Not sure if we still care, but I do run the tests for IE5, IE7, and IE8.